### DISCUSSION TITLE
How to export figures without displaying them?

### DISCUSSION THREAD

**Discussion by nyyssont** [mtex-toolbox/mtex#2009] [Ask Anything] [answered]
Hello,

it seems that MTEX figures can be created without displaying them, for export and save purposes. However, I cannot seem to get this functionality to work.

Could someone post an example showing how to create an mtex figure showing e.g. a band contrast map and save it without displaying it?

**Comment by kilir** [comment, marked as answer]
Hi Tuomo,

after pulling the relevant change
```
mtexdata forsterite 
ebsd1= ebsd.gridify
plot(ebsd,ebsd.prop.bc,'Visible','off')
saveFigure('uha.png')
```
does it. I have less success without gridify.

Cheers,
Rüdiger

**Reply by nyyssont** [reply]
Thank you!
